scripts/freebsd/update.sh

```diff
@@ -19,8 +19,10 @@ fi
 
 # Update FreeBSD
 # NOTE: this will fail if there aren't any patches available for the release yet
-env PAGER=/bin/cat $freebsd_update fetch;
-env PAGER=/bin/cat $freebsd_update install;
+if [ "$major_version" -lt 11 ]; then
+  env PAGER=/bin/cat $freebsd_update fetch;
+  env PAGER=/bin/cat $freebsd_update install;
+fi
```

Comment:
is freebsd_update deprecated or is this gated just to work around the failures when there's no new updates?